lib/applicationinsights.json

```diff
@@ -0,0 +1,6 @@
+{
+  "connectionString": "${file:/mnt/secrets/reform-scan/app-insights-connection-string}",
```

Comment:
this log to bulk-scan ?
first in bulk-scan-shared infra you need to create "app-insights-connection-string" secret.

Reply:
yes, it was created and merged https://github.com/hmcts/bulk-scan-shared-infrastructure/pull/272